src/main/resources/quilt.mod.json

```diff
@@ -1,42 +1,36 @@
 {
 	"schema_version": 1,
 	"quilt_loader": {
-		"group": "${maven_group}",
-		"id": "examplemod",
+		"group": "${group}",
+		"id": "example_mod",
 		"version": "${version}",
 		"metadata": {
 			"name": "Mod Name",
 			"description": "A short description of your mod.",
 			"contributors": {
 				"Me!": "Owner"
 			},
-			"icon": "assets/examplemod/icon.png"
+			"license": "ARR",
+			"icon": "assets/example_mod/icon.png"
 		},
 		"intermediate_mappings": "net.fabricmc:intermediary",
 		"entrypoints": {
-			"init": [
-				"com.examplemod.ExampleMod"
-			]
+			"init": "com.example.example_mod.ExampleMod"
 		},
 		"depends": [
 			{
 				"id": "quilt_loader",
-				"versions": ">=${loader_version}"
+				"versions": ">=0.16.0-"
```

Comment:
Why did you remove the expansion of these?

Reply:
Because it would get annoying with mods hard-depending on bugfix versions and versions that would otherwise be perfectly compatible with